docs/development/module/tests.md

```diff
@@ -116,16 +128,6 @@ fn my_runtime_test() {
 }
 ```
 
-## Next Steps
```

Comment:
Please follow the outline of the "next steps" laid out here.

This is one thing I will be firm on keeping consistant (and we have some CSS stuff which checks for these items on pages)

Reply:
I had renamed it `References and Examples` but ok, I'll change it back

Reply:
Sorry I was not clear enough. I mean the WHOLE format.

* Next Steps
   * Learn More
   * Examples
   * References

If you look at EVERY page it follows this pattern...

Reply:
Notice too the action item are all bullets, so forming your actions as bullets too would be best.